Realm/RLMObjectBase.h

```diff
@@ -26,12 +26,6 @@
 @property (nonatomic, readonly, getter = isInvalidated) BOOL invalidated;
 
 - (instancetype)init;
-- (instancetype)initWithObject:(id)object;
-- (instancetype)initWithObject:(id)value schema:(RLMSchema *)schema;
-- (instancetype)initWithObjectSchema:(RLMObjectSchema *)schema;
-
-- (BOOL)isEqualToObject:(RLMObjectBase *)object;
-- (NSArray *)linkingObjectsOfClass:(NSString *)className forProperty:(NSString *)property;
 
 - (id)objectForKeyedSubscript:(NSString *)key;
```

Comment:
it would be nice to also move these out of `RLMObjectBase`, so in Swift 1.2 I can make them return normal optionals, instead of implicitly unwrapped optionals